app-modules/report/src/Filament/Exports/StudentExporter.php

```diff
@@ -114,12 +114,12 @@ public static function getColumns(string $type = ExportColumn::class): array
                 ->label('Count of Alerts')
                 ->counts('alerts')),
             ...array_map(
-                fn (AlertStatus $status): TextColumn | ExportColumn => static::notDefault($type::make("alerts_{$status->value}_count")
+                fn (SystemAlertStatusClassification $status): TextColumn | ExportColumn => static::notDefault($type::make("alerts_{$status->value}_count")
                     ->label("Count of {$status->getLabel()} Alerts")
                     ->counts([
                         "alerts as alerts_{$status->value}_count" => fn (Builder $query) => $query->status($status),
                     ])),
-                AlertStatus::cases(),
+                SystemAlertStatusClassification::cases(),
```

Comment:
This will need to change based on the Feature flag to get the values from the relationship instead.